src/field-accessor.ts

```diff
@@ -239,7 +239,7 @@ export class FieldAccessor<R, V> extends AccessorBase implements IAccessor {
   validate(options?: ValidateOptions): boolean {
     const ignoreRequired = options != null ? options.ignoreRequired : false;
     const ignoreGetError = options != null ? options.ignoreGetError : false;
-    this.setRaw(this.raw, { ignoreRequired });
+    this.validateRaw(this.raw, { ignoreRequired });
```

Comment:
you can experiment with skipping this altogether if (_raw === undefined && !this.required)

Reply:
I've tried this, unfortunately it seemed to break a few assumptions along the way. Tests started failing. I've decided to opt out of this change for now.